src/app/services/bucket/qarnotBucketService.py

```diff
@@ -0,0 +1,26 @@
+from qarnot import connection, bucket
+import os
+
+from app.services.bucket.baseBucketService import BaseBucketService
+from app import config as cfg
+
+
+class QarnotBucketService(BaseBucketService):
+
+    async def upload_file(self, bucket_name: str, bucket_key: str, file_binary: bin):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
+        return await media_bucket.add_file(bucket_name, file_binary, bucket_key)
+
+    async def get_uploaded_file(self, bucket_name: str, bucket_key: str):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
+        return await media_bucket.get_file(bucket_name, bucket_key)
+
+    async def fetch_bucket_files(self, bucket_name: str):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
+        bucket_local_copy_directory = 'bucket_local_copy'
```

Comment:
I don't understand what it does. There is no way to just get the file names? You have to download them?
Anyway, set the ̀bucket_local_copy_directory` as a config variable

Reply:
Reading the documentation, I think we are interested in the `list_files` function.
At least that is how I thought about this `fetch_bucket_files` behavior. Maybe ̀fetch_bucket_filenames` was more appropriate. To discuss with the others on what we want to do.

Reply:
OK I will fix that